kratos/tests/geometries/test_geometry.h

```diff
@@ -28,14 +28,15 @@
 namespace Kratos {
 namespace Testing {
 
+    static std::ofstream out("/dev/null");
```

Comment:
At the end it was working because `std::cout.rdbuf(out.rdbuf());` was evaluated as NULL, which is an accepted buffer which has the same effect as `/dev/null` in linux. As it happens to be what we want im just making `std::cout.rdbuf(nullptr);`

According to [this](http://stackoverflow.com/questions/14845446/how-redirect-stdostream-to-a-file-or-dev-null-in-c-and-linux) it also sets the `std::ios::badbit` which should prevent conversions in the output in case we eventually have any.